### PR TITLE
[FIX] Fix Windows tests I/O error due to TestResults file mounts

### PR DESCRIPTION
**Pull request by Tarun-Arora**
Please prefix your pull request with one of the following: **[FEATURE]** **[FIX]** **[IMPROVEMENT]**.

**In raising this pull request, I confirm the following (please check boxes):**

- [x] I have read and understood the [contributors guide](https://github.com/CCExtractor/sample-platform/blob/master/.github/CONTRIBUTING.md).
- [x] I have checked that another pull request for this purpose does not exist.
- [x] I have considered, and confirmed that this submission will be valuable to others.
- [x] I accept that this submission may not be used, and the pull request closed at the will of the maintainer.
- [x] I give this submission freely, and claim no ownership to its content.

**My familiarity with the project is as follows (check one):**

- [ ] I have never used the project.
- [ ] I have used the project briefly.
- [ ] I have used the project extensively, but have not contributed previously.
- [x] I am an active contributor to the project.

---

This PR identifies the following error that occurred during Test Runs, specifically on the windows platform, due to rclone mounts...
Screenshot_20221016_115629

FIX: On the Windows platform, we can copy the TestResults folder before tests start; since the size of the folder is small (216 MB at present), it would be feasible.
Screenshot_20221016_115843

### CODE REVIEW

File: install/ci-vm/ci-windows/ci/runCI.bat [hunk 1]
```diff
@@ -9,13 +9,13 @@ if NOT EXIST "variables.bat" (
 
 echo Loading variables.bat
 rem Source variables
-call %~dp0\variables.bat
+call %~dp0\variables.bat
```
Comment (thealphadollar) [inline install/ci-vm/ci-windows/ci/runCI.bat:12]: Please remove the added whitespace.

File: install/ci-vm/ci-windows/ci/runCI.bat [hunk 2]
```diff
@@ -25,12 +25,13 @@ if EXIST "%dstDir%\ccextractorwinfull.exe" (
...
     call :executeCommand "%tester%" --entries "%testFile%" --executable "ccextractorwinfull.exe" --tempfolder "%tempFolder%" --timeout 3000 --reportfolder "%reportFolder%" --resultfolder "%resultFolder%" --samplefolder "%sampleFolder%" --method Server --url "%reportURL%"
 
-    curl -s -A "%userAgent%" --form "type=logupload" --form "file=@%logFile%" -w "\n" "%reportURL%"
+    curl -s -A "%userAgent%" --form "type=logupload" --form "file=@%logFile%" -w "\n" "%reportURL%" >> "%logFile%"
     timeout 10
 
     echo Done running tests
     call :postStatus "completed" "Ran all tests"
 
+    timeout 60
```
Comment (thealphadollar) [inline install/ci-vm/ci-windows/ci/runCI.bat:34]: A timeout of 60, may I know the reason?

**Review by thealphadollar — COMMENTED** [review]
Looks good, just two small considerations.

File: install/ci-vm/ci-windows/ci/runCI.bat (hunk 1 above)
Comment (Tarun-Arora) [reply install/ci-vm/ci-windows/ci/runCI.bat:12]: Yes, Sure.

File: install/ci-vm/ci-windows/ci/runCI.bat (hunk 2 above)
Comment (Tarun-Arora) [reply install/ci-vm/ci-windows/ci/runCI.bat:34]: I tested this before making changes in `controllers.py`. I would test with lower time and make changes if required.

Comment (Tarun-Arora) [reply install/ci-vm/ci-windows/ci/runCI.bat]: Timeout is already present in `postStatus` function, hence removed this timeout.

**Review by thealphadollar — APPROVED** [review]

**Pull request merged**